config/stack_mac.yaml

```diff
@@ -169,13 +169,13 @@ nceppost:
   build: YES
   version: dceca26
   install_as: dceca26
-  openmp: ON
+  openmp: OFF
 
 wrf_io:
   build: YES
   version: v1.1.1-cmake-v5
   install_as: 1.1.1
-  openmp: ON
+  openmp: OFF
```

Comment:
Why do we turn off OpenMP here?

Reply:
No, real reason for wrf_io or post. I'll selectively re-enable it once I get this to build properly.

Clang on macOS doesn't support OpenMP, so it should be disabled for wgrib2 (C code). But somehow OpenMP_C was being propagated to wgrib2 because ip2 or sp are using it resulting in a build failure. OpenMP is marked as found through the CMake config for ip2 when wgrib2 searches for it (it runs find_package(OpenMP) for Fortran), so wgrib2 tries to link to it and it's not available.

Reply:
OK, thanks for the explanation!